README.md

```diff
@@ -1,31 +1,37 @@
 ---
-KILDA CONTROLLER
+OPEN KILDA SDN CONTROLLER
 ---
 [![Build Status](https://github.com/telstra/open-kilda/actions/workflows/unittest.yml/badge.svg)](https://github.com/telstra/open-kilda/actions/workflows/unittest.yml)[![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=org.openkilda%3Akilda-parent&metric=alert_status)](https://sonarcloud.io/dashboard?id=org.openkilda%3Akilda-parent)
 
-## Description
+## Introduction
 
-Kilda Controller is Software Defined Network Controller. It was created to manage a set of switches, create connections
-between them (flows) and react to network equipment failures by evacuating flows from damaged points..
+OpenKilda is a Web-Scale Software-Defined Networking controller. OpenKilda is capable of manage traffic on tens of thousands of switches simultaneously, 
+control millions of flows, and provide sub-second network telemetry. OpenKilda provides a variety of features, such as:
+- manual and automatic management of traffic flows: simple flows, Y-flows, mirroring, single-switch flows;
```

Comment:
- manual and automatic management of L2 services: point-to-point flows, Y-flows, etc.;
- advanced features for some types of services: mirroring, data gathering, path pinning, etc.;

Reply:
I added your suggestions.